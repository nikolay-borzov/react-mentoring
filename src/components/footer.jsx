import React from 'react'

import { SiteName } from './site-name'

export function Footer() {
  return (
    <footer className="alt-background">
      <SiteName />
    </footer>
  )
}
