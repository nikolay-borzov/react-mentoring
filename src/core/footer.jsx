import React from 'react'

import { SiteName } from './site-name'

export function Footer() {
  return (
    <footer className="layout-block">
      <SiteName />
    </footer>
  )
}
