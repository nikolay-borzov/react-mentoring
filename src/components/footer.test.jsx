import React from 'react'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'

import { Footer } from './footer'

describe('Footer component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON()

    expect(tree).toMatchSnapshot()
  })

  it(`contains 'SiteName' component`, () => {
    const wrapper = shallow(<Footer />)

    expect(wrapper.find('SiteName').length).toBe(1)
  })
})
