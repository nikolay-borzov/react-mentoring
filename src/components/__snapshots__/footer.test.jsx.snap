// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders correctly 1`] = `
<footer
  className="alt-background padding-controls"
>
  <span
    className="site-name"
  >
    Movie Search
  </span>
</footer>
`;
